xlators/storage/posix/src/posix-inode-fd-ops.c

```diff
@@ -5833,9 +5833,9 @@ posix_fill_readdir(fd_t *fd, struct posix_fd *pfd, off_t off, size_t size,
     return count;
 }
 
-dict_t *
-posix_entry_xattr_fill(xlator_t *this, inode_t *inode, fd_t *fd,
-                       char *entry_path, dict_t *dict, struct iatt *stbuf)
+static dict_t *
+posix_entry_xattr_fill(xlator_t *this, inode_t *inode, char *entry_path,
```

Comment:
function signature is changed (`fd_t *fd` is removed). Not used in other places?

Reply:
Sorry I missed this. Please ignore this comment.